10/collection-meta.yaml

```diff
@@ -287,6 +288,8 @@ collections:
       - rndmh3ro
       - schurzi
     repository: https://github.com/T-Systems-MMS/ansible-collection-icinga-director
+  vmware.vmware:
+    repository: https://github.com/ansible-collections/vmware.vmware
```

Comment:
Can we hold this PR until we get at least one maintainer to list here?

Reply:
The inclusion request comes with a list of GH handles:

`GitHub Handles: @machacekondra @mikemorency @bardielle @mariolenz`

These should be added as maintainers.